asm/istio/options/citadel-ca.yaml

```diff
@@ -28,8 +28,8 @@ spec:
     trustDomain: "cluster.local"
     defaultConfig:
       proxyMetadata:
-        GKE_CLUSTER_URL: "" # {"$ref":"#/definitions/io.k8s.cli.substitutions.gke-cluster-url"}
-        GCP_METADATA: "" # {"$ref":"#/definitions/io.k8s.cli.substitutions.gke-metadata"}
+        GKE_CLUSTER_URL: ""
+        GCP_METADATA: ""
```

Comment:
I bet the right thing to do is actually remove these two completely from citadel-ca and move them to off-gcp without the refs.

Reply:
ok. would you say the same about **GCE_METADATA_HOST** field too (expand citadel-ca.yaml)?

Reply:
Yep, good catch

Reply:
done